src/System.Runtime.CompilerServices.VisualC/src/System/Runtime/CompilerServices/Attributes.cs

```diff
@@ -57,9 +54,6 @@ public static partial class IsUdtReturn
     {
     }
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
-    public static partial class IsVolatile
```

Comment:
The type is being removed, but the attribute that was applied to it wasn't deleted?